compiler/src/main/scala/org/scalajs/core/compiler/Compat210Component.scala

```diff
@@ -100,6 +100,43 @@ trait Compat210Component {
     }
   }
 
+  // Compat to support: new overridingPairs.Cursor(sym).iterator
+
+  implicit class OverridingPairsCursor2Iterable(cursor: overridingPairs.Cursor) {
+    def iterator: Iterator[SymbolPair] = new Iterator[SymbolPair] {
+
+      skipIgnoredEntries()
+
+      def hasNext: Boolean = cursor.hasNext
+
+      def next(): SymbolPair = {
+        val low = cursor.overriding
+        val high = cursor.overridden
+        cursor.next()
+        skipIgnoredEntries()
+        new SymbolPair(low, high)
+      }
+
+      private def skipIgnoredEntries(): Unit = {
+        while (ignoreCurrentEntry)
+          cursor.next()
+      }
+
+      private def ignoreCurrentEntry: Boolean =
+        cursor.overriding.name == nme.isInstanceOf_
+    }
+
+    class SymbolPair(val low: Symbol, val high: Symbol)
+
+    /** To make this compat code compile in 2.11 as the fields `overriding` and
+     *  `overridden` are only present in 2.10.
+     */
+    private implicit class Cursor210toCursor211(cursor: overridingPairs.Cursor) {
+      def overriding: Symbol = throw new UnsupportedOperationException
+      def overridden: Symbol = throw new UnsupportedOperationException
```

Comment:
Use `sys.error("infinite loop in Compat")`, like at other similar places in this file.